tests/queries/0_stateless/01923_network_receive_time_metric_insert.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
-# Tags: no-fasttest
+# Tags: no-fasttest, no-parallel
 # Tag no-fasttest: needs pv
+# Tags: no-parallel: reads from a system table
```

Comment:
I think here you meant to type`Tag` instead of `Tags`. Otherwise, it'd follow the same syntax as the first line which sets the tags to be applied, not the description of why the tag is included

Reply:
fixed, thanks